src/components/Photography/PhotographyGridSection.tsx

```diff
@@ -3,148 +3,141 @@ import 'react-image-lightbox/style.css'
-        {isLightboxOpen && lightboxSrc && (
-          <Lightbox
-            enableZoom={false}
-            mainSrc={lightboxSrc}
-            nextSrc={nextImageSrc}
-            prevSrc={prevImageSrc}
-            onCloseRequest={this.closeLightbox}
-            onMovePrevRequest={this.decrementLightboxIndex}
-            onMoveNextRequest={this.incrementLightboxIndex}
-          />
-        )}
-      </StyledPanel>
-    )
-  }
-
-  private decrementLightboxIndex = () => {
-    this.toggleLightbox({ index: this.state.index - 1, isLightboxOpen: true })
-  }
-
-  private incrementLightboxIndex = () => {
-    this.toggleLightbox({ index: this.state.index + 1, isLightboxOpen: true })
-  }
-
-  private closeLightbox = () => {
-    this.toggleLightbox({ ...this.state, isLightboxOpen: false })
-  }
-
-  private clickImageElement = (imageIndex: number) => {
-    this.toggleLightbox({
-      index: imageIndex,
-      isLightboxOpen: !this.state.isLightboxOpen,
-      lightboxImages: this.state.lightboxImages,
-    })
-  }
-
-  private toggleLightbox = ({
-    index = this.state.index,
-    isLightboxOpen = this.state.isLightboxOpen,
-    lightboxImages = this.state.lightboxImages,
-  }: ToggleLightboxOptions) => {
-    // prettier-ignore
-    this.setState({ // lgtm [js/react/inconsistent-state-update]
+  // Measure the width of the container element.
+  // @ts-ignore
+  const [ref, { width }] = useDimensions()
+
+  // Form a grid of stacked images, using the width & column calculations we got
+  // from the `useMedia` and `useDimensions` hooks above.
+  const heights: number[] = new Array(columns).fill(0) // Each column gets a height starting with zero
+  const gridItems = sortedImages.map((child, index) => {
+    const { childImageSharp } = child
+    const imageWidth = child.width || width / columns
+    const aspectRatio = _.get(childImageSharp, 'sizes.aspectRatio')
+    const height = _.isFinite(child.height)
+      ? child.height
+      : imageWidth * (2 / aspectRatio)
+    // Masonry-grid placing — positions each tile sequentially into the
+    // smallest column available.
+    const column = heights.indexOf(Math.min(...heights))
+
+    const xy = [
+      (width / columns) * column,
+      (heights[column] += height / 2) - height / 2,
+    ]
+    return {
+      ...child,
+      columns,
+      height: height / 2,
       index,
-      isLightboxOpen,
-      lightboxImages,
-      lightboxSrc: lightboxImages[index],
-      nextImageSrc: lightboxImages[(index + 1) % lightboxImages.length],
-      prevImageSrc: lightboxImages[(index - 1) % lightboxImages.length],
-    })
-  }
+      width: width / columns,
+      xy,
+    }
+  })
+
+  const transitions = useTransition(gridItems, fp.get('id'), {
+    from: ({ xy, width, height }) => ({ xy, width, height }),
+    // enter: ({ xy, width, height }) => ({ xy, width, height }),
+    update: ({ xy, width, height }) => ({ xy, width, height }),
+    config: { mass: 5, tension: 500, friction: 100 },
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The key 'config' is not sorted alphabetically](https://app.codacy.com/app/jdstuart/jesses.io/pullRequest?prid=4576429)